envelope.js

```diff
@@ -1,3 +1,4 @@
+// TODO: remove this file? if it's replaced by box2
```

Comment:
todo

Reply:
@mixmix this is a good idea right? box2 gives all this functionality, and better encapsulates it/separates the logic from this module

Reply:
@mixmix

Reply:
SWEET